src/main/java/ttt/CommandLineGameController.java

```diff
@@ -15,18 +16,26 @@
 import static ttt.ReplayOption.Y;
 
 public class CommandLineGameController {
+    private final GameConfiguration gameConfiguration;
     private GameRules gameRules;
     private Prompt gamePrompt;
+    private GameType gameType;
 
-    public CommandLineGameController(GameRules gameRules, Prompt commandLinePrompt) {
+
+    public CommandLineGameController(GameConfiguration gameConfiguration, GameRules gameRules, Prompt commandLinePrompt) {
```

Comment:
This constructor is only used for tests, so that the GameRules are setup. (Otherwise you would always have to call playMove() so that it had been programatically setup).